.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build images
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'eureka-relayer-v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Are these suppose to be our release tags for the relayer? How about `release/relayer-v[0-9]+.[0-9]+.[0-9]+` or `relayer-v[0-9]+.[0-9]+.[0-9]+`? Since I think we want to eventually rebrand IBC Eureka to just IBC, and this repo would become the new IBC repo.

I don't feel strongly about this so you can leave as is if you feel strongly about this.

Reply:
I think it would be nice to keep a consistent tag scheme from ibc-go.
In ibc-go we tag builds on main with just "main" (similar to latest). For manual dispatch builds we typically tag with a git hash, which I also quite like.

We should probably have a different tag for an actual release version, which we can probably do later, and that would just be in the format `vX.Y.Z`

Reply:
https://github.com/cosmos/relayer/pkgs/container/relayer we already have a relayer package (cosmos/relayer)

Reply:
i'll let you make the decision on what you want the versioning to be, i'll use this for nwo